src/composables/useTable.ts

```diff
@@ -165,11 +171,101 @@ export const useTable = () => {
     }
   };
 
+  const sortByTeam = (
+    rows: readonly TableRow[],
+    sortBy: string,
+    descending: boolean,
+  ): readonly TableRow[] => {
+    const data = [...rows];
+
```

Comment:
Maybe you can split this func into sub functions...

Reply:
Updated using the if - else if structure.